### PR TITLE
Validation: descriptions

### PR DESCRIPTION
**Pull request by SueADarby**
start of next 30

### CODE REVIEW

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-pipmodeproperty-field-microsoft-web-media-smoothstreaming.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: SmoothStreamingMediaElement.PipModeProperty Field (Microsoft.Web.Media.SmoothStreaming)
+description: The PipModeProperty Field represents a dependency property that specifies the PiPMode()()()() property. It is a dependency based on Microsoft.Web.Media.SmoothStreaming.
```
Comment (chrisms86) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-pipmodeproperty-field-microsoft-web-media-smoothstreaming.md:3]: Are the parentheses necessary here?

**Comment by SueADarby** [comment]
**Global effort to fix build validation errors**
@Rick-Anderson  @mcleblanc 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Comment by Rick-Anderson** [comment]
@SueADarby fix conflict

**Comment by SueADarby** [comment]
> @SueADarby fix conflict

Fixed!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**